route/build/src/historyFile.f90

```diff
@@ -327,45 +296,6 @@ SUBROUTINE closeNC(this)
       endif
     END SUBROUTINE closeNC
 
-    ! ---------------------------------
```

Comment:
It's nice to not to HAVE to call of these as new files are created. However, you could keep this code around and call it only when mizuRoute is closing down.

Reply:
So now that pioSys are pio decomposition descriptors) are not in the histFile type, I think it would be better to clean function outside this type??   I have clean function in pio_utits.f90

Reply:
Ahh, OK, should be removed.